### DISCUSSION TITLE
Cambio zona molto lento con iPhone 15

### DISCUSSION THREAD

**Discussion by mirtillo64** [indomus/forum#3169] [GENERALE - Discussioni generiche]
Buongiorno, ho un problema che non riesco a risolvere, uso home Assistant da 3/4 mesi, non sono ancora molto pratico ma qualcosa ho imparato.
20 giorni fa ho cambiato cellulare, sono passato dal mio vecchio iPhone X al 15 , con il vecchio avevo cominciato a creare delle automomzioni accensioni e spegnimenti ad orari prestabiliti (luci e tapparelle) poi ho cominciato a testare con una tapparella automazioni basate su entra e esci dalla zona di casa e funzionava.
Purtroppo da quando ho cambiato cell , le automazioni più semplici funzionano ancora, però quelle basate sulle zone funzionano male.
Nel senso che funzionano ma con ritardi di 15/20 minuti , ho impostato una zona casa di 200 metri, quando esco o entro la mappa associata a me ci mette parecchio ad aggiornarsi.
Scusate per il lungo post, ma non riesco ad uscirne non capisco se il problema è home Assistant o il cellulare.
Grazie per un eventuale aiuto saluti

**Comment by KC-inDomus** [comment]
hai impostato "precise location" nel menu gps dell'app companion?

**Reply by mirtillo64** [reply]
Si

**Comment by mirtillo64** [comment]
Buongiorno, nessuno a un’idea di cosa possa essere ?

**Reply by Franco2255** [reply]
ciao, problema simile con iphone xn con automazione apertura carraio quando arrivo in zona casa, la cosa che ho notato è dove si trova il cellulare al momento dell'ingresso nella zona (100 metri) se è in tasca o non riceve il segnale o lo riceve dopo che sono entrato ina casa, se  è sul cruscotto della macchina alcune volte funzione correttamente appena entro in zona altre volte ritarda specialmente se è nuvoloso 
boooooooooo

**Reply by mirtillo64** [reply]
Ciao a me è indifferente se in tasca o in mano,
speriamo che qualcuno riesca a darci una dritta

**Comment by mirtillo64** [comment]
Ciao risolto il problema ,
Ora funziona ho ricliccato aggiungi server dall’ app complementare e ora funziona.
Mi ha creato un doppione dei vari sensori, ho escluso i vecchi e funziona.
Ma mi è sorto un’altro problema, l’automazione che uso nella tapparella delle mia camera, la ho impostata al rientro apri 45% e funziona ma se apro dalla Dashboard o dal pulsante fisico si apre solo al 45% .
In pratica anche manualmente prende il comando dall’ automazione.
Mi sto scoraggiando

**Comment by mirtillo64** [comment]
Risolto
